src/category_theory/sites/sieves.lean

```diff
@@ -332,9 +332,9 @@ begin
 end
 
 /-- A sieve induces a presheaf. -/
-@[simps]
+-- @[simps]
```

Comment:
Why are you removing this `@[simps]`. Is it generating unhelpful lemmas? I guess that we need some simp-lemmas of some sort, right?

Reply:
Accidental, thanks!